ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -339,6 +346,9 @@ func (r *RayJobReconciler) Reconcile(ctx context.Context, request ctrl.Request)
 		return ctrl.Result{RequeueAfter: RayJobDefaultRequeueDuration}, nil
 	}
 
+	// check for retries and update deployment status to Retrying
+	r.checkBackoffLimitAndUpdateStatusIfNeeded(ctx, rayJobInstance)
+
 	// This is the only place where we update the RayJob status. Please do NOT add any code
 	// between the above switch statement and the following code.
```

Comment:
nit: Please do NOT add any code between `checkBackoffLimitAndUpdateStatusIfNeeded` and the following code.

Reply:
done, but is the comment really relevant anymore?